### PR TITLE
feature/rebranding/frontend-task

### PR DESCRIPTION
**Pull request by ervinpiol**

### CODE REVIEW

File: web/src/app/admin/users/PedingInvites.tsx
Comment (SchadenKai) [inline web/src/app/admin/users/PedingInvites.tsx:1]: keep the current changes from the development env for now if there are no changes. The development env already have an attached endpoint to it

File: web/src/app/admin/users/PedingInvites.tsx [hunk 1]
```diff
@@ -154,16 +105,19 @@ export const PendingInvites = ({ q }: { q: string }) => {
...
+                              <span className="text-sm text-subtle truncate max-w-44">
+                                {user.email}
                               </span>
                             </div>
                           </div>
                         </TableCell>
                         <TableCell>
                           <div className="flex gap-2 justify-end">
-                            {/* <Button>Resend Invite</Button> */}
+                            <Button>Resend Invite</Button>
```
Comment (SchadenKai) [inline web/src/app/admin/users/PedingInvites.tsx:120]: comment this out first

File: web/src/app/admin/users/AllUsers.tsx [hunk 2]
```diff
@@ -336,7 +336,7 @@ export const AllUsers = ({ q }: { q: string }) => {
         <div className="text-sm pt-2 pb-4 space-y-2">
           <p>
             Anyone with an email address with any of the following domains can
-            sign up: vividsolution.io,enmedd.com,arnold.io.
+            sign up: vividsolution.io, enmedd.com, arnold.io.
```
Comment (SchadenKai) [inline web/src/app/admin/users/AllUsers.tsx:339]: the emails here are dynamic based on the `validDomains: string[]` data.
there is also a default value of message if there are no `validDomains` set

**Review by SchadenKai — CHANGES_REQUESTED** [review]

File: web/src/app/admin/users/PedingInvites.tsx (hunk 1 above)
Comment (ervinpiol) [reply web/src/app/admin/users/PedingInvites.tsx]: done

File: web/src/app/admin/users/AllUsers.tsx (hunk 2 above)
Comment (ervinpiol) [reply web/src/app/admin/users/AllUsers.tsx]: done

File: web/src/app/admin/users/PedingInvites.tsx
Comment (ervinpiol) [reply web/src/app/admin/users/PedingInvites.tsx:1]: done

**Review by SchadenKai — APPROVED** [review]

**Pull request merged**